src/main/java/org/jitsi/jibri/capture/ffmpeg/executor/FfmpegExecutor.kt

```diff
@@ -0,0 +1,43 @@
+package org.jitsi.jibri.capture.ffmpeg.executor
+
+import org.jitsi.jibri.sink.Sink
+import org.jitsi.jibri.util.MonitorableProcess
+
+/**
+ * Paramaters which will be passed to ffmpeg
+ */
+data class FfmpegExecutorParams(
+    val resolution: String = "1280x720",
+    val framerate: Int = 30,
+    val videoEncodePreset: String = "veryfast",
+    val queueSize: Int = 4096,
+    val streamingMaxBitrate: Int = 2976,
+    val streamingBufSize: Int = streamingMaxBitrate * 2,
+    // The range of the CRF scale is 0–51, where 0 is lossless,
+    // 23 is the default, and 51 is worst quality possible. A lower value
+    // generally leads to higher quality, and a subjectively sane range is
+    // 17–28. Consider 17 or 18 to be visually lossless or nearly so;
+    // it should look the same or nearly the same as the input but it
+    // isn't technically lossless.
+    // https://trac.ffmpeg.org/wiki/Encode/H.264#crf
+    val h264ConstantRateFactor: Int = 25,
+    val gopSize: Int = framerate * 2
```

Comment:
Are these defaults or hardcoded values? If hardcoded, do you ever foresee that being an issue that's tricky to overcome?

Reply:
these are hard-coded.  the old one had these hard-coded, so i didn't feel the need to handle being able to dynamically set these values until we come across that use case.